src/app/components/utils/footer/footer.component.html

```diff
@@ -1 +1,26 @@
-<p>footer works!</p>
+<div class="text-center" id="main">
+    <!--Footer Elements -->
+    <h5 class="text-center">URL Storage</h5> 
+    
+   <div class="container">
+       <ul class="list-unstyled list-inline text-center">
+           <li class="list-inline-item">
+             <a href="#" class="text-dark"><i class="fab fa-facebook"></i></a>
```

Comment:
Let's have GitHub, yoututbe, instagram, linkedin in this order. What do you think?